protected/humhub/modules/user/models/fieldtype/UserEmail.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\user\models\fieldtype;
+
+use humhub\libs\Html;
+
+/**
+ * UserEmail is a virtual profile field
+ * that displays the current email address of the user.
+ *
+ * @since 1.6
+ */
+class UserEmail extends BaseTypeVirtual
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function getUserValue($user, $raw = true)
+    {
+        if (empty($user->email)) {
+            return '';
+        }
+
+        if ($raw) {
+            return Html::encode($user->email);
```

Comment:
Shouldn't raw return an unencoded value?

Reply:
It's always encoded but without any extras like Link tags.
https://github.com/humhub/humhub/blob/master/protected/humhub/modules/user/models/fieldtype/Text.php#L203